index.js

```diff
@@ -63,46 +73,42 @@ var chromeProfileToCallgrind = function(profile, outStream) {
         call.selfTime += node.selfTime;
 
         var childCalls = call.childCalls;
-        if (node.children) {
-            node.children.forEach(function(child) {
-                var childUID = child.callUID;
-                var childCall = childCalls[childUID] = childCalls[childUID] || {
-                    functionName: child.functionName,
-                    url: child.url,
-                    totalHitCount: 0,
-                    totalTime: 0,
-                    lineNumber: child.lineNumber
-                };
-                childCall.totalHitCount += child.totalHitCount;
-                childCall.totalTime += child.totalTime;
-            });
+        for(var j = 0; j < node.children.length; j++) {
```

Comment:
IIRC, this `node.children` check was in place because `.children` can be entirely missing sometimes, so `node.children.length` would throw `Cannot access property length of undefined`